src/Particles/babylon.particle.ts

```diff
@@ -20,6 +70,15 @@
             other.size = this.size;
             other.angle = this.angle;
             other.angularSpeed = this.angularSpeed;
+            other.particleSystem = this.particleSystem;
+            other.cellIndex = this.cellIndex;
+            other._loopAnimation = this._loopAnimation;

```

Comment:
These info can be read from the particle system. No need to duplicate them on all particles. (Trying to save memory here)